data/categories.txt

```diff
@@ -7118,6 +7118,45 @@ sw:Mbuga
 fa:پارک
 mr:उद्यान|पार्क
 
+tourism-aquarium
```

Comment:
`tourism-aquarium|@tourism` ?
to add into the "Sights" search category

Reply:
Added